app/views/posts/_comments.html.erb

```diff
@@ -1,35 +1,26 @@
-<% comments.each do |comment| %>
-  <div class = "comment" >
-    <div class="card p-8 mb-1">
-      <div id="vote-actions-<%= @post.id %>" class="vote" data-id="<%= @post.id %>">
-        <div class="fa fa-arrow-up upvote <%= is_upvoted @post %>"></div>
-        <span class="font-weight-bold score"><%= @post.score %></span>
-        <div class="fa fa-arrow-down downvote <%= is_downvoted @post %>"></div>
-
-        <% if comment.account.profile_image.attached? %>
-          <%= link_to image_tag(comment.account.profile_image, width: "30px" , height:"30px", :class => "small-pic float-left m-r-15"),profile_path(comment.account) %>
-        <% else %>
-          <img class="small-pic float-left m-r-15" width="30px" height="30px" src="https://www.iconpacks.net/icons/2/free-reddit-logo-icon-2436-thumb.png">
-        <% end %>
-        <strong><%= link_to comment.account.full_name, profile_path(comment.account.username), class: "text-dark" %></strong>
-        <%= comment.message %>
-        <div class = "fl">
-          <p class = "text-muted m-l-30"><%= time_ago_in_words comment.created_at %> ago </p>
-          <% if !(@post.closed?) %>
-            <% if signed_in? %><a href = "#" class="comment-form-display">Reply</a><% end %>
-          <% end %>
-          <div class = "comment-form">
-            <%= render partial: "comments/form", locals: {post: comment.post, parent: comment, comment_id: comment.id}%>
-          </div>
-        </div>
+<div class = "comment">  
+  <% comments.each do |comment| %>
+    <% if comment.account.profile_image.attached? %>
+      <%= link_to image_tag(comment.account.profile_image, width: "30px" , height:"30px", :class => "small-pic float-left m-r-15"),profile_path(comment.account) %>
```

Comment:
still in some of the places this old syntax is used.please fix this

:class=>

Reply:
I have changed the syntax